client/components/settings/local_settings_modal.html

```diff
@@ -0,0 +1,21 @@
+<div class="ui right floated basic icon compact item" ng-click="localSettingsModal.modalVisible = true; localSettingsModal.loadSettings()">
```

Comment:
Using the `sm-modal-show` as an attribute does not require using the `sm-button` as an element. You can put it on the `menu item` and it would work and look nice.